Gordon360/Services/ProfileService.cs

```diff
@@ -301,6 +301,48 @@ public async Task UpdateMobilePrivacyAsync(string username, string value)
             _context.SaveChanges();
         }
 
+        /// <summary>
+        /// update planned graduation
+        /// </summary>
+        /// <param name="username">The username for the user whose planned graduation year is to be updated</param>
+        /// <param name="newPlannedGraduationYear">The new year to update the user's planned graduation year to</param>
+        public async Task<StudentProfileViewModel> UpdatePlannedGraduationYearAsync(string username, string newPlannedGraduationYear)
+        {
+            var profile = GetStudentProfileByUsername(username);
+            if (profile == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found" };
+            }
```

Comment:
```suggestion
            var profile = GetStudentProfileByUsername(username);
            if (profile == null)
            {
                throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found" };
            }
```

Why do we get the get the profile here if we don't use it? Also I don't know how this table is scripted if at all, but in the event a student doesn't have a CUSTOM_PROFILE row, what happens?

Reply:
fixed and tested, thanks!